index.html

```diff
@@ -80,19 +82,21 @@ <h4 class="features-title">Сувениры</h4>
             </li>
           </ul>
         </li>
-        <li class="advantages-item">
+        <li class="advantages-item-picture">
```

Comment:
Лучше было бы сделать вторым классом:
```suggestion
        <li class="advantages-item advantages-item-picture">
```

Reply:
исправила